### DISCUSSION TITLE
FileNotFoundError: File not found: "affine"

### DISCUSSION THREAD

**Discussion by shathaa1983** [TorchIO-project/torchio#976] [General]
Hi, 

I am trying to apply torchio on 3D medical images segmentation, on the training stage I got this error:

FileNotFoundError: File not found: "affine".
![image](https://user-images.githubusercontent.com/69587655/194396564-89849fd9-9188-4c26-a724-b2cdf6139a41.png)

Could you please advise?

**Comment by romainVala** [comment]
hello
we can not advise anything without further information 
try to give more context with a reproducible example

**Reply by fepegar** [reply]
@shathaa1983, I suggest you follow the instructions at https://github.com/fepegar/torchio/issues/new?assignees=&labels=&template=not_working.yml

**Reply by shathaa1983** [reply]
Hi,
I followed the instructions. The new issue is open right now.

Thanks
